tools/snap/snapcraft.yaml

```diff
@@ -0,0 +1,54 @@
+name: syzkaller
```

Comment:
How users are supposed to discover this file? Shouldn't it be some known file name? Or it's tools/snap/snapcraft.yaml in all repositories?